src/Chunk.cpp

```diff
@@ -2637,6 +2637,32 @@ void cChunk::GetBlockTypeMeta(int a_RelX, int a_RelY, int a_RelZ, BLOCKTYPE & a_
 
 
 
+void cChunk::GetBlockTypeMeta(Vector3i a_Location, BLOCKTYPE & a_BlockType, NIBBLETYPE & a_BlockMeta) const
+{
+	int RelX = a_Location.x - GetPosX() * cChunkDef::Width;
+	int RelZ = a_Location.z - GetPosZ() * cChunkDef::Width;
+
+	GetBlockTypeMeta(RelX, a_Location.y, RelZ, a_BlockType, a_BlockMeta);
+}
+
+
+
+
+
+void cChunk::GetBlockTypeMeta(Vector3d a_Location, BLOCKTYPE & a_BlockType, NIBBLETYPE & a_BlockMeta) const
```

Comment:
It just saves calling `floor()` when dealing the monster, whose vectors are mostly Vector3d. It is *that* necessary but why not?